inst/@sym/private/op_helper.m

```diff
@@ -44,19 +43,20 @@
 
   % note: cmd is already cell array, hence [ concatenates with it
   cmd = [ cmd
-          '(x, y) = _ins'
-          'if x.is_Matrix and y.is_Matrix:'
-          '    assert x.shape == y.shape'
-          '    A = sp.Matrix(x.shape[0], x.shape[1],'
-          '        lambda i,j: _op(x[i,j], y[i,j]))'
-          '    return A,'
-          'elif x.is_Matrix:'
-          '    return x.applyfunc(lambda a: _op(a, y)),'
-          'elif y.is_Matrix:'
-          '    return y.applyfunc(lambda a: _op(x, a)),'
-          'else:'
-          '    return _op(x, y),' ];
+          'l = (1, 1)'
+          'for i in range(len(_ins)):'
+          '    if isinstance(_ins[i], MatrixBase):'
+          '        if _ins[i].shape != (1, 1):'
+          '            l = _ins[i].shape'
+          '        else:'
+          '            assert l == _ins[i].shape, "Matrix must have equal sizes"'
+          '    else:'
+          '        _ins[i] = Matrix([_ins[i]])'
+          'base = zeros(*l)'
+          'for i in itertools.product(*map(Range, l)):'
+          '    base.__setitem__(i, _op(*[k[0] if k.shape == (1, 1) else k.__getitem__(i) for k in _ins]))'
+          'return base,' ];
 
-  z = python_cmd (cmd, sym(x), sym(y));
+  z = python_cmd (cmd, sym(varargin){:});
```

Comment:
Hi, i split the chain, about cellfun sadly i can't use it, at least in a right way, because it converts the cells to matrix! and with it we can't unpack it to a function. Obvs i could researched bad, if you know some way please tell me and i'll update this.
In some way i prefer do this in other pr, because i think we should update all the code with that instead only a few lines.

If don't exist, maybe use a helper instead?

Thx. Cya.